documentation/troubleshooting.md

```diff
@@ -28,3 +40,196 @@ Account: [myname@example.org]
+
+CassKop make uses of the PodDisruptionBudget to prevent CassKop to make some change on the CassandraCluster that could
+make more than 1 Cassandra node at a time. 
+
+If you have a Pod stuck in **pending** state, then you have at least 1 Pod in Disruption, and the PDB object will
+prevent you to make changes on statefulset because that mean that you will have more than 1 Cassandra down at a time.
+
+```console
+$ k get poddisruptionbudgets
+NAME             MIN AVAILABLE   MAX UNAVAILABLE   ALLOWED DISRUPTIONS   AGE
+cassandra-demo   N/A             1                 0                     12m
+```
+
+The Operator logs this line when there is disruption on the Cassandra cluster:
+
+```logs
+INFO[3037] Cluster has Disruption on Pods, we wait before applying any change to statefulset  cluster=cassandra-demo dc-rack=dc1-rack1
+```
+
+
+### Example: can't ScaleUp
+
+In this example I ask a ScaleUp but it can't perform :
+
+```console
+$ k get pods
+NAME                                                              READY   STATUS    RESTARTS   AGE
+cassandra-demo-dc1-rack1-0                                        1/1     Running   0          16h
+cassandra-demo-dc1-rack1-1                                        0/1     Pending   0          12m
+cassandra-demo-dc1-rack2-0                                        1/1     Running   0          16h
+cassandra-demo-dc1-rack3-0                                        1/1     Running   0          16h
+```
+
+the cassandra-demo-dc1-rack1-1 pod is Pending and can't be scheduled.
+
+If we looked at the pod status we will see this message :
+
+```
+Warning  FailedScheduling   51s (x17 over 14m)    default-scheduler   0/6 nodes are available: 4 Insufficient cpu, 4 node(s) didn't match node selector.
+```
+
+Kubernetes can't find any Pod with sufficient cpu and matching kubernetes nodes labels we asked in the topology section.
+
+To fix this, we can either:
+- reduce memory/cpu limits
+- add more kubernetes nodes that will satisfied our requirements.
+- rollback the scaleUp Operation
+
+> At this point, CassKop will wait indefinitely to the case to be Fix
+
+
+#### Rollback ScaleUp operation
+
+In order to rollback the operation, we need to revert the change on the `nodesPerRacks` parameter.
+
+> This is not sufficient
+
+Because CassKop is actually performing another action on the cluster (ScaleUp) we can't scheduled a new operation to
+rollback since it has not finished.
+We introduced a new parameter in the CRD to allow such changes when all the pods can't be scheduled:
+- `Spec._unlockNextOperation: true`
+
+> _**:triangular_flag_on_post: Warning** This is not a regular parameter and it must be used with very good care!!._
+
+
+By adding this parameter in our cluster definition, CassKop will allow to trigger a new operation.
+
+> Once CassKop will scheduled the new operation, it will reset the `Spec._unlockNextOperation: false` to the default false
```

Comment:
@allamand Once CassKop `has` scheduled the new operation, it will reset this parameter to the default `false` value